### DISCUSSION TITLE
Win32Exception: No process is on the other end of the pipe

### DISCUSSION THREAD

**Discussion by nitinanand27** [episerver/Foundation#868] [General]
On starting website, i get following error. Any idea how to solve this?
![image](https://user-images.githubusercontent.com/15047308/186665157-5f435475-8838-49fb-ae6a-6c2db61c9fa2.png)

**Comment by OptimizelyMVP** [comment]
In SQL Server Configuration Manager, make sure you have the 'Named Pipes' protocol enabled for the SQL Server Database Engine that contains the databases for the site.
